encrypt-core/encrypt-core-merge/src/main/java/org/apache/shardingsphere/encrypt/merge/dal/impl/EncryptColumnsMergedResult.java

```diff
@@ -66,19 +70,50 @@ public final boolean next() throws SQLException {
         return true;
     }
     
+    private Collection<String> getAssistedQueryColumns() {
+        return getTableEncryptColumnMetaDatas().stream().map(EncryptColumnMetaData::getAssistedQueryColumnName)
+                .collect(Collectors.toList());
+    }
+    
+    private Collection<String> getPlainColumns() {
+        return getTableEncryptColumnMetaDatas().stream().map(EncryptColumnMetaData::getPlainColumnName)
+                .collect(Collectors.toList());
+    }
+    
+    private Collection<EncryptColumnMetaData> getTableEncryptColumnMetaDatas() {
+        Collection<EncryptColumnMetaData> result = new LinkedList<>();
+        for (Map.Entry<String, ColumnMetaData> entry : schemaMetaData.get(tableName).getColumns().entrySet()) {
```

Comment:
Please do not use `Map.Entry`, just use `Entry`

Reply:
OK, I fix it. Thanks.